### PR TITLE
Added the php tags to prevent issues when using multiple php versions

### PR DESCRIPTION
**Pull request by amasolini**
Added the "php" word to the php tags to avoid errors when installing the project on a system with the short_open_tag directive disabled.

As per the documentation (http://php.net/manual/en/language.basic-syntax.phptags.php), the use of short tags is not recommended and we should have them.

### CODE REVIEW

**Comment by spastorelli** [comment]
LGTM on my end

File: fetch.php [hunk 1]
```diff
@@ -1,5 +1,7 @@
-<?
+<?php
```
Comment (spastorelli) [inline fetch.php:1]: Looks like `fetch.php` is missing the closing tag too.

**Review by spastorelli — APPROVED** [review]
LGTM on my end. On minor comment.

File: fetch.php (hunk 1 above)
Comment (amasolini) [reply fetch.php:1]: End tags are not required anymore unless you are inserting a php snippet in the middle of another file (like an html file).

**Pull request closed without merging**